crates/turbopath/src/relative_unix_path.rs

```diff
@@ -0,0 +1,21 @@
+use std::path::PathBuf;
+
+use path_slash::PathBufExt;
+
+use crate::RelativeSystemPathBuf;
+
+pub struct RelativeUnixPath<'a> {
+    inner: &'a str,
+}
+
+impl<'a> From<&'a str> for RelativeUnixPath<'a> {
```

Comment:
Yeah this is basically an unchecked construction, which IMO should be explicit and not the default behavior

Reply:
Agreed. Thinking about it more, we should not have this impl and it should be explicit.